Description:Update help menu item and add about menu items for different distributions
Author: ?

Index: gnome-panel-2.30.0/gnome-panel/panel-menu-items.c
===================================================================
--- gnome-panel-2.30.0.orig/gnome-panel/panel-menu-items.c	2010-03-09 15:34:06.000000000 +1100
+++ gnome-panel-2.30.0/gnome-panel/panel-menu-items.c	2010-06-03 13:57:49.626446947 +1000
@@ -1028,6 +1028,7 @@
 	GtkWidget *item;
 	char      *gconf_name;
 	char      *name;
+	char      *path;
 	char      *uri;
 	GFile     *file;
 
@@ -1194,9 +1195,18 @@
 	if (add_separator)
 		add_menu_separator (menu);
 
-	panel_menu_items_append_from_desktop (menu, "yelp.desktop", NULL, FALSE);
+	panel_menu_items_append_from_desktop (menu, "yelp.desktop", _("Help and Support"), FALSE);
+
 	panel_menu_items_append_from_desktop (menu, "gnome-about.desktop", NULL, FALSE);
 
+	if (g_file_test (DATADIR"/omf/about-ubuntu/about-ubuntu-C.omf",
+			 G_FILE_TEST_IS_REGULAR))
+	  panel_menu_items_append_from_desktop (menu, "ubuntu-about.desktop", NULL, FALSE);
+
+	if (g_file_test (DATADIR"/omf/about-edubuntu/about-edubuntu-C.omf",
+			 G_FILE_TEST_IS_REGULAR))
+	  panel_menu_items_append_from_desktop (menu, "edubuntu-about.desktop", NULL, FALSE);
+
 	if (parent->priv->append_lock_logout)
 		panel_menu_items_append_lock_logout (menu);
 }
